.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        julia-version: ['1.3', '1', 'nightly']
+        julia-version: ['1', 'nightly']
```

Comment:
We should add 1.6 because 1 means 1.7, yet Project.toml allows 1.6.

Reply:
yeah fixed in #44